dspace-api/src/test/java/org/dspace/app/csv/CSVMetadataImportReferenceIT.java

```diff
@@ -300,14 +386,17 @@ public void testNonUniqueRowName() throws Exception {
     @Test(expected = MetadataImportException.class)
     public void testNonUniqueMDRefInDb() throws Exception {
         context.turnOffAuthorisationSystem();
-        ItemBuilder.createItem(context, col1)
-                   .withRelationshipType("Person")
-                   .withIdentifierOther("1")
-                   .build();
-        ItemBuilder.createItem(context, col1)
-                   .withRelationshipType("Person")
-                   .withIdentifierOther("1")
-                   .build();
+        WorkspaceItem workspaceItem = workspaceItemService.create(context, col1, false);
+        Item person = installItemService.installItem(context, workspaceItem);
+        itemService.addMetadata(context, person, "relationship", "type", null, null, "Person");
+        itemService.addMetadata(context, person, "dc", "identifier", "other", null, "1");
+        itemService.update(context, person);
+        WorkspaceItem workspaceItem2 = workspaceItemService.create(context, col1, false);
+        Item person2 = installItemService.installItem(context, workspaceItem2);
+        itemService.addMetadata(context, person2, "relationship", "type", null, null, "Person");
+        itemService.addMetadata(context, person2, "dc", "identifier", "other", null, "1");
+        itemService.update(context, person2);
```

Comment:
Same here, revert to using Builders again.  I'll stop repeating this message now.  Essentially, **MOST** of your changes to this IT class can now be reverted. You can use Builders here as the Builder classes are now available in `dspace-api`.